simpeg/electromagnetics/frequency_domain/sources.py

```diff
@@ -723,6 +723,93 @@ def bPrimary(self, simulation):
         return mkvc(b)
 
 
+class ElectricDipole(BaseFDEMSrc):
+    r"""
+    Point electric dipole source.
+
+
+    Parameters
+    ----------
+    receiver_list : list of simpeg.electromagnetics.frequency_domain.receivers.BaseRx
+        A list of FDEM receivers
+    frequency : float
+        Source frequency
+    location : (dim) numpy.ndarray, default: numpy.r_[0., 0., 0.]
+        Source location.
+    strength : float, default: 1.0
+        Source strength (Am).
+    orientation : {'z', x', 'y'} or (dim) numpy.ndarray
```

Comment:
Following [numpydoc's style guide](https://numpydoc.readthedocs.io/en/latest/format.html#parameters), let's put the default value first:
```suggestion
    orientation : {'x', y', 'z'} or (dim) numpy.ndarray
```

Reply:
Good point, done!